src/backend/marsha/core/management/commands/sync_medialive_video.py

```diff
@@ -0,0 +1,101 @@
+"""Check every existing medialive channel and check if the related video is still usable."""
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from marsha.core.defaults import IDLE, RUNNING, STOPPED
+from marsha.core.models import Video
+from marsha.core.utils.medialive_utils import list_medialive_channels, update_id3_tags
+from marsha.core.utils.time_utils import to_timestamp
+
+
+class Command(BaseCommand):
+    """
+    Once a live started, video states becomes indirectly linked to medialive channel state.
+    Their is a risk that sometimes, states are not sync for an unknown reason,
+    for example, the api was down when aws was calling our callback to update video states.
+    This can let us with media channel "IDLE", while in our side, the video state is "RUNNING"
+    In this case, the video becomes soft locked, the possible actions for a "RUNNING"
+    video will fail because the medialive channel state is "IDLE"
+    This command iterates on medialive channel and ensure that linked videos are
+    state synced to avoid this kind of soft lock.
+    """
+
+    help = __doc__
+
+    def is_channel_sync_with_video(self, video_state, channel_state):
```

Comment:
```suggestion
    def is_channel_sync_with_video(self, live_state, channel_state):
```

and for `channel_state` you are using `medialive_state` in the calling function. You should be consistent